akka-docs/rst/java/distributed-data.rst

```diff
@@ -240,6 +240,15 @@ Subscribers will receive ``Replicator.DataDeleted``.
 
 .. includecode:: code/docs/ddata/DistributedDataDocTest.java#delete
 
+.. warning::
+
+  As deleted keys continue to be included in the stored data on each node, as well as
+  in gossip messages that are sent out when new nodes joins the cluster, a continuous series of
```

Comment:
I think you can remove the "that are sent out when new nodes joins the cluster", because the deleted keys are included in all `Status` messages that are part of the gossip protocol, i.e. it's not only when syncing new nodes.